themes/default/content/docs/aws/ec2.md

```diff
@@ -20,29 +20,87 @@ The `@pulumi/aws` library enables fine-grained control over the AWS EC2 resource
 ```javascript
 const aws = require("@pulumi/aws");
 
-const eip = new aws.ec2.Eip("myeip");
-
-const securityGroup = new aws.ec2.SecurityGroup("mysecuritygroup", {
-    ingress: [
-        { protocol: "tcp", fromPort: 80, toPort: 80, cidrBlocks: ["0.0.0.0/0"] },
-    ],
+// Create a VPC.
+const vpc = new aws.ec2.Vpc("vpc", {
+    cidrBlock: "10.0.0.0/16",
 });
 
-const vpc = new aws.ec2.Vpc("myvpc", {
-    cidrBlock: "10.0.0.0/16"
-})
+// Create an an internet gateway to give the instance access to the internet.
+const internetGateway = new aws.ec2.InternetGateway("internet-gateway", {
+    vpcId: vpc.id,
+});
 
-const internetGateway = new aws.ec2.InternetGateway("myinternetgateway", {
+// Create a public subnet.
+const subnet = new aws.ec2.Subnet("subnet", {
     vpcId: vpc.id,
+    cidrBlock: "10.0.1.0/24",
+    mapPublicIpOnLaunch: true,
 });
 
-const publicRouteTable = new aws.ec2.RouteTable("myroutetable", {
+// Create a route table.
+const routes = new aws.ec2.RouteTable("routes", {
+    vpcId: vpc.id,
     routes: [
         {
             cidrBlock: "0.0.0.0/0",
             gatewayId: internetGateway.id,
         },
     ],
+});
+
+// Associate the route table with the public subnet.
+const routeTableAssociation = new aws.ec2.RouteTableAssociation("route-table-association", {
+    subnetId: subnet.id,
+    routeTableId: routes.id,
+});
+
+// Create an elastic IP for the public subnet.
+const elasticIP = new aws.ec2.Eip("elastic-ip", {
```

Comment:
This doesn't do anything...

Reply:
Oh, ha, look at that. 😄 Thanks, copy-pasta.